roles/preview_common/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
This would probably be the place to add the HELM_OPTS/extra files handling, if we do that in this round